src/features/components/templates/why-trade-options/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import Container from 'features/components/atoms/container'
+import Typography from 'features/components/atoms/typography'
+import { Localize } from 'components/localization'
+import Button from 'features/components/atoms/button'
+import Flex from 'features/components/atoms/flex-box'
+import TradeItem, { TradeItemType } from 'features/components/molecules/trade-item'
+import { TString } from 'types/generics'
+import { LinkUrlType } from 'features/types'
+import LinkButton from 'features/components/atoms/link-button'
+
+export type WhyTradeOptionsType = {
+    heading: TString
+    heading2: TString
+    button: {
+        text: TString
+        url: LinkUrlType
+    }
+    items: TradeItemType[]
+}
+
+type WhyTradeOptionsProps = {
+    data: WhyTradeOptionsType
+}
+
+const WhyTradeOptions = ({ data }: WhyTradeOptionsProps) => {
```

Comment:
We can use React.PropsWithChildren or second variant to type data instead of creating new type
```suggestion
const WhyTradeOptions = ({ data }: PropsWithChildren<WhyTradeOptionsType> || { data: WhyTradeOptionsType }) => {
```

Reply:
Done...